diff --git a/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js b/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
index f4f7e81..483b6df 100644
--- a/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
+++ b/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
@@ -68,12 +68,6 @@ export default class AnimatedEllipsis extends Component {
   }
 }
 
-AnimatedEllipsis.propTypes = {
-  numberOfDots: PropTypes.number,
-  animationDelay: PropTypes.number,
-  minOpacity: PropTypes.number,
-  style: Text.propTypes.style
-};
 AnimatedEllipsis.defaultProps = {
   numberOfDots: 3,
   animationDelay: 300,
